compass/ocean/mesh/remap_topography.py

```diff
@@ -93,65 +107,151 @@ def run(self):
         """
         Run this step of the test case
         """
+        self._create_target_scrip_file()
+        self._partition_scrip_file('source.scrip.nc')
+        self._partition_scrip_file('target.scrip.nc')
+        self._create_weights()
+        self._remap_to_target()
+        self._modify_remapped_bathymetry()
+
+    def _create_target_scrip_file(self):
+        """
+        Create target SCRIP file from MPAS mesh file.
+        """
+        logger = self.logger
+        logger.info('Create source SCRIP file')
+
         config = self.config
+        section = config['remap_topography']
+        expandDist = section.getfloat('expandDist')
+        expandFactor = section.getfloat('expandFactor')
+
+        descriptor = MpasCellMeshDescriptor(
+            fileName='base_mesh.nc',
+            meshName=self.mesh_name,
+        )
+        descriptor.to_scrip(
+            'target.scrip.nc',
+            expandDist=expandDist,
+            expandFactor=expandFactor,
+        )
```

Comment:
Before this call, we want to make sure to set `descriptor.format = 'NETCDF3_64BITDATA'` (Check this, it's just my best guess on the names of the attribute and its value.)